gcmt/utils/interface.py

```diff
@@ -205,23 +209,24 @@ def chem_from_poorman(self, temp_key='T', CO=0.55, FeH=0.0):
         temp_key: str, optional
             The key to the temperature field used for the abundancies.
             Defaults to T.
-        CO: float, optional
+        co_ratio: float, optional
             The C/O ratio. Currently only one global value allowed. Defaults to 0.55.
-        FeH: float, optional
+        feh_ratio: float, optional
             The metalicity ratio. Currently only one global value allowed. Defaults to 0.0.
         """
-        if self.ds is None:
+        if self.dsi is None:
             raise ValueError("Data is missing. Use interface.set_data() first.")
 
-        return self.chemistry.chem_from_poorman(temp_key=temp_key, CO=CO, FeH=FeH)
+        return self.chemistry.chem_from_poorman(temp_key=temp_key, co_ratio=co_ratio,
+                                                feh_ratio=feh_ratio)
 
 
-class pRTInterface(Interface):
+class PrtInterface(Interface):
```

Comment:
Needs to be reflected in the docs as well. I think the docs currently generate docs for `pRTInterface`